packages/core/.babelrc

```diff
@@ -1,5 +1,6 @@
 {
   "plugins": ["annotate-pure-calls"],
+  "presets": ["@babel/preset-typescript"],
```

Comment:
Reason for this change (open for better solutions):
When I start to use `makeStyles` in devtools, I get the following error during build. And the error only goes away when I add babel TS preset in both @griffel/core and @griffel/react
<img width="788" alt="Screenshot 2022-04-19 at 12 45 06" src="https://user-images.githubusercontent.com/28751745/164010418-e00053fe-1e67-46c0-8952-3d53259b58ec.png">

Reply:
We agreed on a clearer option:

```
{
  "plugins": ["annotate-pure-calls"],
  "env": {
    "development": {
      "presets": ["@nrwl/react/babel"]
    },
    "test": {
      "presets": ["@nrwl/react/babel"]
    }
  }
}
```